src/Assign/README.md

```diff
@@ -0,0 +1,111 @@
+# Assign
+```haskell
+Assign Object
+```
+`Assign` is a `Monoid` that will combine two objects using `Object.assign` on their `Object` values, merging and overwriting new values for values that are not undefined.
+
+```js
+const Assign = require('crocks/Assign')
+
+const first = { name : "Bob" }
```

Comment:
Are we using a linter?

Reply:
The lininting is not enforced on the READMEs...yet.
Will look into that, think this will do it for us:
https://github.com/eslint/eslint-plugin-markdown

PR coming...